application/src/main/kotlin/net/corda/v5/application/messaging/FlowSession.kt

```diff
@@ -12,17 +12,18 @@ import net.corda.v5.base.types.MemberX500Name
  *
  * There are two ways of obtaining such a session:
  *
- * 1. Calling [FlowMessaging.initiateFlow]. This will create a [FlowSession] object on which the first send/receive operation will attempt
- *    to kick off a corresponding [InitiatedBy] flow on the counterparty's node.
+ * 1. Calling [FlowMessaging.initiateFlow]. This will create a [FlowSession] object on which the first send/receive
+ * operation will attempt to kick off a corresponding [InitiatedBy] flow on the counterparty's node.
  *
- * 2. As constructor parameter to [InitiatedBy] flows. This session is the one corresponding to the initiating flow and may be used for
- *    replies.
+ * 2. As constructor parameter to [InitiatedBy] flows. This session is the one corresponding to the initiating flow
+ * and may be used for replies.
  */
 @DoNotImplement
 interface FlowSession {
```

Comment:
I think we should include the same example from `FlowMessaging` here as well? Or should we link to `FlowMessaging` using `@see`?